src/ClassyResource/_makeRequest.js

```diff
@@ -25,13 +25,20 @@ export default function _makeRequest(path, method, headers, form, data) {
   }
 
   const promise = new Promise((resolve, reject) => {
+    // the encoding param defaults to undefined, which means the response will be stringified
+    // we want to preserve the binary encoding for pdfs, so return null in this case
+    const encoding = _.includes(headers['Content-Type'], 'pdf')
```

Comment:
If our use case here is _receiving_ a binary PDF file in response to a GET request, it seems like we might want to check the `headers` collection for an `Accept` header of `application/pdf`, rather than `Content-Type` (which would imply that we're uploading a PDF).

Reply:
good call, updated!